Workbooks/WorkbooksMetadata.json

```diff
@@ -1,5 +1,18 @@
 [
-    {
+  {
+    "workbookKey": "AttackSurfaceReduction",
+    "logoFileName": "M365securityposturelogo.svg",
+    "description": "This workbook helps you implement the ASR rules of Windows/Defender, and to monitor them over time. The workbook can filter on ASR rules in Audit mode and Block mode.",
+    "dataTypesDependencies": [ "DeviceEvents"],
+    "dataConnectorsDependencies": [ "MicrosoftThreatProtection" ],
+    "previewImagesFileNames": [ "AttackSurfaceReductionWhite.png", "AttackSurfaceReductionBlack.png" ],
+    "version": "1.0.0",
+    "title": "Attack Surface Reduction Dashboard",
+    "templateRelativePath": "AttackSurfaceReduction.json",
+    "subtitle": "",
+    "provider": "Daniel Chronlund"
```

Comment:
Hey @DanielChronlund, provider should be name of organization, can you please check and update, thanks.

Reply:
@v-sabiraj, this is a private MVP contribution. Is a company name required as the provider?

Reply:
@DanielChronlund, you can add as Microsoft or Microsoft Sentinel community

Reply:
> @DanielChronlund, you can add as Microsoft or Microsoft Sentinel community